stone/backends/python_types.py

```diff
@@ -640,15 +640,22 @@ def _generate_custom_annotation_processors(self, ns, data_type, extra_annotation
         dt, _, _ = unwrap(data_type)
         if is_struct_type(dt) or is_union_type(dt):
             annotation_types_seen = set()
-            for _, annotation in dt.recursive_custom_annotations:
-                if annotation.annotation_type not in annotation_types_seen:
-                    yield (annotation.annotation_type,
-                           generate_func_call(
-                               'bb.make_struct_annotation_processor',
-                               args=[class_name_for_annotation_type(annotation.annotation_type, ns),
-                                     'processor']
-                           ))
-                    annotation_types_seen.add(annotation.annotation_type)
+            if is_struct_type(dt) and dt.has_enumerated_subtypes():
```

Comment:
I'm not entirely sure what these blocks are doing...  Would be great to leave some comments here describing what it's trying to do.